xml/Microsoft.Vsa/IVsaItem.xml

```diff
@@ -92,8 +92,8 @@
         <ReturnType>Microsoft.Vsa.VsaItemType</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
+        <summary>Gets the specified object's type, as determined by the [IVsaItems.CreateItem Method](https://msdn.microsoft.com/en-us/library/bb385458\(v=vs.90\)) method.</summary>
```

Comment:
This could be an xref, no? /<xref:Microsoft.Vsa.IVsaItems.CreateItem%2A?displayProperty=nameWithType>

Reply:
cref actually. I missed this one while reviewing the msdn links in summaries.